### DISCUSSION TITLE
DWMBlurGlass Keeps crashing at random moments

### DISCUSSION THREAD

**Discussion by ugohohcohxihxihdig** [Maplespe/DWMBlurGlass#165] [Q&A]
So I have a issue where I'm using DWMBlurGlass it often crashes and disables aero Which is annoying could you please fix this issue because I'm tired of it

**Comment by ALTaleX531** [comment]
First, we need to know your system version as well as the build number, second we need to know how we can reproduce the crash, and then we need to know your current configuration. Without providing this information we can't do anything and we won't do anything.

**Comment by ugohohcohxihxihdig** [comment]
> First, we need to know your system version as well as the build number, second we need to know how we can reproduce the crash, and then we need to know your current configuration. Without providing this information we can't do anything and we won't do anything.



my system version is Windows 11 2H23 build 22621.nirelease.220506-1250 and This problem happens randomly when screen flickers and whole aero disables (Also this problem happened only in Windows 10 When I was using custom theme https://www.deviantart.com/vaporvance/art/Aero10-for-Windows-10-1903-22H2-909711949)
